src/routes.json

```diff
@@ -98,11 +93,17 @@
         "title": "DMCA"
     },
     {
-        "name": "camp",
-        "pattern": "^/camp/?$",
-        "routeAlias": "/camp/?$",
-        "view": "camp/camp",
-        "title": "Down Deep"
+        "name": "donate",
```

Comment:
I think that, if `store` is gonna be in the redirect, this probably should be too. Or there's the option of moving `store` into the main views list. I'd be fine with either option, whatever you think is best @TheGrits !